axon-framework/queries/query-dispatchers.md

```diff
@@ -13,11 +13,12 @@ The `QueryBus` is the mechanism that dispatches queries to query handlers. Queri
 
 The `QueryGateway` is a convenient interface towards the query dispatching mechanism. While you are not required to use a gateway to dispatch queries, it is generally the easiest option to do so. It abstracts certain aspects for you, like the necessity to wrap a Query payload in a Query Message.
 
-Regardless whether you choose to use the `QueryBus` or the `QueryGateway`, both provide several types of queries. Axon Framework makes a distinction between three types, being:
+Regardless whether you choose to use the `QueryBus` or the `QueryGateway`, both provide several types of queries. Axon Framework makes a distinction between four types, being:
```

Comment:
The streaming query stuff is for 4.6.0, not 4.5.
As such, it doesn't belong in this PR.

@schananas added these to `master` since `master` will become the 4.6.0. branch.
I guess you've updated `4.5` with `master` though.